plugin/environment.go

```diff
@@ -448,13 +449,17 @@ func (env *Environment) RunMultiPluginHook(hookRunnerFunc func(hooks Hooks) bool
 	})
 }
 
-// SetPrepackagedPlugins saves prepackaged plugins in the environment
+// SetPrepackagedPlugins saves prepackaged plugins in the environment.
 func (env *Environment) SetPrepackagedPlugins(plugins []*PrepackagedPlugin) {
 	env.prepackagedPluginsLock.Lock()
 	env.prepackagedPlugins = plugins
 	env.prepackagedPluginsLock.Unlock()
 }
 
+func (env *Environment) GetPrepackagedPlugins() []*PrepackagedPlugin {
```

Comment:
I added a getter too in my PR and now it's merged into the branch. Could you update and we can converge on using the same one?

Reply:
Updated.